From: Cyril Brulebois <cyril@debamax.com>
Date: Fri, 22 Jan 2021 13:25:54 +0000
Subject: Adjust systemd unit

 - Drop PIDFile (that uses an obsolete path, and doesn't seem to be
   used at all).
 - Adjust paths for the packaged crowdsec binary (/usr/bin).
 - Drop commented out ExecStartPost entirely.
 - Drop syslog.target dependency, it's socket-activated (thanks to the
   systemd-service-file-refers-to-obsolete-target lintian tag).

--- a/config/crowdsec.service
+++ b/config/crowdsec.service
@@ -1,14 +1,12 @@
 [Unit]
 Description=Crowdsec agent
-After=syslog.target network.target remote-fs.target nss-lookup.target
+After=network.target remote-fs.target nss-lookup.target
 
 [Service]
 Type=notify
 Environment=LC_ALL=C LANG=C
-PIDFile=/var/run/crowdsec.pid
-ExecStartPre=/usr/local/bin/crowdsec -c /etc/crowdsec/config.yaml -t
-ExecStart=/usr/local/bin/crowdsec -c /etc/crowdsec/config.yaml
-#ExecStartPost=/bin/sleep 0.1
+ExecStartPre=/usr/bin/crowdsec -c /etc/crowdsec/config.yaml -t
+ExecStart=/usr/bin/crowdsec -c /etc/crowdsec/config.yaml
 ExecReload=/bin/kill -HUP $MAINPID
 
 [Install]
